content/en/monitors/monitor_types/forecasts.md

```diff
@@ -86,29 +69,37 @@ The "reactive" model more easily extrapolates recent behavior, at the risk of ov
 
 {{< img src="monitors/monitor_types/forecasts/linear_reactive.png" alt="linear reactive" responsive="true" style="width:80%;" >}}
 
-The "default" model is what Goldilocks would choose, and adjusts to the most recent trend and extrapolate that line, while being robust to recent noise.
+The "default" model is the middle of the road choice which adjusts to the most recent trend. It extrapolates a line while being robust to recent noise.
 
 {{< img src="monitors/monitor_types/forecasts/linear_default.png" alt="linear default" responsive="true" style="width:80%;">}}
 
-**Seasonal:** Use this algorithm for seasonal metrics. In monitors, Datadog auto-detects the seasonality of the metric and choose between weekly, daily, and hourly seasonality. This algorithm requires at least 2 seasons of history for it to start forecasting, and potentially uses up to 6.
+**Seasonal:** Use this algorithm for seasonal metrics. In monitors, Datadog auto-detects the seasonality of the metric and chooses between weekly, daily, and hourly seasonality. This algorithm requires at least two seasons of history for it to start forecasting, and potentially uses up to six.
 
 Examples of seasonality options:
 
-* **weekly**: the algorithm expects that this Monday will behave like past Mondays.
-* **daily**: the algorithm expects that 7pm today wlil be have like 7pm in past days.
-* **hourly**: the algorithm expects that 7:15 will behave like 6:15, 5:15, 4:15, etc.
+* **weekly**: the algorithm expects this Monday behaves like past Mondays.
+* **daily**: the algorithm assumes that 7 pm today is like 7 pm from previous days.
+* **hourly**: the algorithm expects that 7:15 behaves like 6:15, 5:15, 4:15, etc.
 
 {{< img src="monitors/monitor_types/forecasts/seasonal.png" alt="seasonal" responsive="true" style="width:80%;">}}
 
 ### Accessing Advanced Options
 
-The advanced options are exposed under the **Advanced** tab in the **New Monitor** page. To specify them in the dashboards (using the JSON tab) or in the API, the format is as follows.
+Access advanced options under the **Advanced** tab in the **New Monitor** page. To specify them in the dashboards (using the JSON tab) or in the API, use this format:
 
 For Linear: `forecast(metric_name, 'linear', 1, interval='60m', history='1w', model='default')`, where the options for `model` are: `default`, `simple`, and `reactive`.
 
 For Seasonal: `forecast(metric_name, 'seasonal', 1, interval='60m', seasonality='weekly')`, where the options for `seasonality` are: `hourly`, `daily`, and `weekly`.
 
-The start and end times to specify when using the API are the start and end times of the forecast itself. If you want the forecast for the next day you would specify the start to be `now` and the end to be `1 day ahead`.
+When using the API, specify the start and end times of the forecast itself. If you want the forecast for the next day, specify the start to be `now` and the end to be `1 day ahead`.
+
+## Forecast dashboards and notebooks
```

Comment:
I would never consider a dashboard a "forecast dashboard" or a notebook a "forecast notebook." How about calling this section"Forecasting in dashboards and notebooks"?